### ISSUE TITLE
Geting "IndexError: list index out of range"

### ISSUE THREAD

**Issue by masiv1001** [AbdelrhmanNile/UnderTaker141#7]
**Describe the bug**
The app crashes after opening whenever the qBittorrent client is open (or if it has some torrent downloading). It also crashes when trying to download any game, sending the download command to qBittorrent before closing.
I've tried with both flatpak and rpm versions of qBittorrent

**To Reproduce**
1. Try download any game.
2. Crash

**Expected behavior**
It shouldn't crash, dead simple.

**Desktop (please complete the following information):**
 - OS: Fedora 38
 
EDIT: Running latest version 2.1.1 released 3 hours ago at the time of writing.

**Additional context**
```
[DEBUG  ] Login successful
[DEBUG  ] [http          ]//localhost:8080 "POST /api/v2/torrents/info HTTP/1.1" 200 826
 Traceback (most recent call last):
   File "/tmp/.mount_UnderTJ1Xx82/opt/src/app.py", line 19, in <module>
     UnderTaker141().run()
   File "/tmp/.mount_UnderTJ1Xx82/opt/python3.11/lib/python3.11/site-packages/kivy/app.py", line 955, in run
     self._run_prepare()
   File "/tmp/.mount_UnderTJ1Xx82/opt/python3.11/lib/python3.11/site-packages/kivy/app.py", line 925, in _run_prepare
     root = self.build()
            ^^^^^^^^^^^^
   File "/tmp/.mount_UnderTJ1Xx82/opt/src/app.py", line 16, in build
     return MainScreen(__version__)
            ^^^^^^^^^^^^^^^^^^^^^^^
   File "/tmp/.mount_UnderTJ1Xx82/opt/src/widgets/core/core.py", line 56, in __init__
     self.load_screens()
   File "/tmp/.mount_UnderTJ1Xx82/opt/src/widgets/core/core.py", line 126, in load_screens
     screen = screen_class(name=plugin.name, qbt_client=self.qbt_client) # create instance of a screen class
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "/tmp/.mount_UnderTJ1Xx82/opt/src/screens/library.py", line 37, in __init__
     self.load_library()
   File "/tmp/.mount_UnderTJ1Xx82/opt/src/screens/library.py", line 48, in load_library
     self.layout.add_widget(GameLibraryCard(game_torrent=game, qbt_client=self.qbt_client))            
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "/tmp/.mount_UnderTJ1Xx82/opt/src/widgets/game/game.py", line 85, in __init__
     self.game_obj = self.query_game(self.game_name)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   File "/tmp/.mount_UnderTJ1Xx82/opt/src/widgets/game/game.py", line 183, in query_game
     return db.get_game(partial_name)[0]
            ~~~~~~~~~~~~~~~~~~~~~~~~~^^^
 IndexError: list index out of range
```

**Comment by masiv1001** [comment]
> which game caused this?

Fear & Hunger, but it crashes before executing it (I should first research on their dependencies I know), it crashes right after hitting the download button.

**Comment by AbdelrhmanNile** [comment]
@masiv1001 
torrent names had html encodings for symbols such as `&` encoded as `&amp;`
fixed in `[v2.1.2](https://github.com/AbdelrhmanNile/UnderTaker141/releases/tag/latest)`
